components/library/arpack/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2016 Sergey Avseyev
+#
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		arpack
+COMPONENT_VERSION=	3.3.0
+REAL_NAME=		arpack-ng
+COMPONENT_PROJECT_URL=	https://github.com/opencollab/arpack-ng
+COMPONENT_SUMMARY=	Collection of Fortran77 subroutines designed to solve large scale eigenvalue problems
+COMPONENT_DESCRIPTION=	The package is designed to compute a few eigenvalues and corresponding eigenvectors of a general n by n matrix A. It is most appropriate for large sparse or structured matrices A where structured means that a matrix-vector product w <- Av requires order n rather than the usual order n**2 floating point operations. This software is based upon an algorithmic variant of the Arnoldi process called the Implicitly Restarted Arnoldi Method (IRAM).
+COMPONENT_FMRI=         library/arpack
+COMPONENT_CLASSIFICATION=       System/Libraries
+COMPONENT_SRC=		$(REAL_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_VERSION).tar.gz
```

Comment:
Keep the default archive name as $(COMPONENT_NAME)-$(COMPONENT_VERSION).tar.gz for storage and only set the $(COMPONENT_VERSION).tar.gz in the URL.